src/test/java/ui/LoginTest.java

```diff
@@ -54,27 +57,27 @@ public void testNegativeLoginNoPass() {
         this.loginPage.open();
         this.loginPage.verify();
         this.loginPage.loginEmail(emailValid);
-        String noPassErrorMessage = this.loginPage.getNoPassErrorMessage();
-        Assert.assertEquals(noPassErrorMessage, this.loginPage.getExpectedLoginNoPassErrMsg(), "The error message is " +
+        String noPassErrorMessage = this.loginPage.getErrorMessage();
+        Assert.assertEquals(noPassErrorMessage, this.loginPage.getExpectedLoginErrMsg("noPassword"), "The error message is " +
                 "not correct.");
     }
 
     @Test
     public void testNegativeLoginNoEmailNoPass() {
         this.loginPage.open();
         this.loginPage.verify();
-        this.loginPage.login();
-        String noUserErrorMessage = this.loginPage.getNoUserErrorMessage();
-        Assert.assertEquals(noUserErrorMessage, this.loginPage.getExpectedLoginNoUserErrMsg(), "The error message is " +
+        this.loginPage.clickLoginButton();
+        String noUserErrorMessage = this.loginPage.getErrorMessage();
+        Assert.assertEquals(noUserErrorMessage, this.loginPage.getExpectedLoginErrMsg("noUser"), "The error message is " +
                 "not correct.");
     }
 
     @Test
-    public void testPositiveLogin() {
+    public void testPositiveLogin() throws InterruptedException {
```

Comment:
Is this exception needed?

Reply:
no, it's not needed, remains of an old code.